ngraph/frontend/paddlepaddle/src/op/hard_sigmoid.cpp

```diff
@@ -0,0 +1,34 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "hard_sigmoid.hpp"
+#include <ngraph/opsets/opset6.hpp>
+#include <paddlepaddle_frontend/utility.hpp>
+
+namespace ngraph
+{
+    namespace frontend
+    {
+        namespace pdpd
+        {
+            namespace op
+            {
+                NamedOutputs hard_sigmoid(const NodeContext& node)
+                {
+                    auto data = node.get_ng_input("X");
+                    float slope = node.get_attribute<float>("slope", 0.2f);
+                    float offset = node.get_attribute<float>("offset", 0.5f);
+                    auto alpha =
+                        ngraph::opset6::Constant::create(ngraph::element::f32, {}, {slope});
```

Comment:
Dont think it is a good idea to hard code types here. The type must match type of data.

Reply:
@luo-cheng2021 Could you retrieve the data type from `get_element_type` instead of using hard-coded `float32` ?

Reply:
Updated in 51cf188800224d7d624c28bf43cd39abe6d8428c.